apps/app/src/hooks/useCloseOnBreakpoint.ts

```diff
@@ -0,0 +1,28 @@
+import { Dispatch, SetStateAction, useCallback, useEffect, useState } from "react";
+
+type UseCloseOnBreakpointProps = {
+	initialState: boolean;
+	breakpoint: number;
+};
+
+export const useCloseOnBreakpoint = ({
+	initialState,
+	breakpoint,
+}: UseCloseOnBreakpointProps): [boolean, Dispatch<SetStateAction<boolean>>] => {
+	const [isOpen, setIsOpen] = useState(initialState);
+
+	const handleResize = useCallback(() => {
+		if (window.innerWidth > breakpoint) {
```

Comment:
Może się mylę, ale wydaje mi się, że w tailwindzie breakpoint jest od różnych 640px
```suggestion
		if (window.innerWidth >= breakpoint) {
```